clicommand/pipeline_upload.go

```diff
@@ -262,7 +263,9 @@ var PipelineUploadCommand = cli.Command{
 		}
 
 		result, err := cfg.parseAndInterpolate(src, input, environ)
-		if err != nil {
+		if w := warning.As(err); w != nil {
+			l.Warn("There were some issues with the pipeline input:\n%v", w)
```

Comment:
is it worth adding something like `pipeline upload will proceed, but may not succeed`, or something?